legacy_packages/wallets/src/evm/connectors/wallet-connect/index.ts

```diff
@@ -339,6 +339,11 @@ export class WalletConnectConnector extends WagmiConnector<
       this._provider = await EthereumProvider.init({
         showQrModal: this.showWalletConnectModal,
         projectId: this.options.projectId,
+        methods: [
+          "eth_sendTransaction",
+          "personal_sign",
+          "eth_signTypedData_v4",
```

Comment:
Does v4 not support signTransaction?

Reply:
Most external wallets don't support eth_signTransactions due to security concerns

Either way in this case I'm just force requesting wallet support for eth_signTypedData_v4, if wallets support other methods they'll be available